docs/sidebars.js

```diff
@@ -4,11 +4,16 @@ const sidebars = {
     {
       type: 'doc',
       label: 'What is OpenFGA',
-      id: 'content/authorization-and-openfga',
+      id: 'content/intro',
```

Comment:
assuming these slug changes are/will be handled on the S3 side?

Reply:
@rhamzeh can you help? I think this is handled by docusaurus